nautobot/dcim/filters/__init__.py

```diff
@@ -89,6 +89,7 @@
     SoftwareVersion,
     VirtualChassis,
 )
+from nautobot.dcim.models.device_components import VDC
```

Comment:
I think it might be convention to declare `VDC` device/models/__init__.py and import VDC from there.